packages/flutter/lib/src/material/dialog_theme.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import 'dialog.dart';
+
+/// Defines a theme for [Dialog] widgets.
+///
+/// Descendant widgets obtain the current [DialogTheme] object using
+/// `DialogTheme.of(context)`. Instances of [DialogTheme] can be customized by 
```

Comment:
can be customized width [DialogTheme.copyWith]

Reply:
Done.